docs/user/README.md

```diff
@@ -401,6 +432,23 @@ The wrap command can be used to wrap a given target with a pair of symbols
 - `"round wrap <TARGET>"`: wraps the target with parentheses
 - `"box wrap <TARGET>"`: wraps the target with square brackets
 
+### Slice
+
+The slice command is used to add multiple cursors to the editor. Each cursor is inserted at the same column on each row requested within the command.
```

Comment:
fwiw you can use with things other than "pre", eg "take", "chuck tail air slice bat", etc